src/styles/style.css

```diff
@@ -0,0 +1,250 @@
+body {
+  font-family: Roboto, sans-serif;
+  margin: 0;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.header__logo-img {
+  display: flex;
+}
+
+.nav__list {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  text-transform: uppercase;
+  text-decoration: none;
+  font-weight: 500;
+  font-size: 12px;
+  color: #060b35;
+  font-style: normal;
+  line-height: 60px;
+  display: flex;
+  align-items: center;
+  position: relative;
+}
+
+.is-active,
+.nav__link[data-qa='nav-hover']:hover {
+  color: #00acdc;
+  transition: color 300ms ease;
+}
+
+.is-active::after {
+  content: '';
+  width: 100%;
+  height: 4px;
+  border-radius: 2px;
+  display: block;
+  bottom: 0;
+  position: absolute;
+  background-color: #00acdc;
+}
+
+.card__rate-stars {
+  display: flex;
+}
+
+.card__rate-stars-star {
+  width: 16px;
+  height: 16px;
+  background: url(../images/star.svg) no-repeat center;
```

Comment:
The background property for '.card__rate-stars-star' is using a shorthand that includes 'no-repeat center'. Ensure that this is the intended behavior for the background image.